content/subscription/faq.md

```diff
@@ -46,10 +40,16 @@ You can't pause or delay a subscription. If a subscription invoice hasn't been p
 
 Subscriptions are non-transferable.
 
-### I have more questions regarding the new pricing, who can I reach out to?
+### I have more questions regarding pricing, who can I reach out to?
 
-If you have any questions that are not covered in the FAQ, [please reach out](https://www.docker.com/company/contact) and a member of the Docker team will get back to you.
+If you have any questions that aren't covered here, [reach out](https://www.docker.com/company/contact) and a member of the Docker team will get back to you.
 
-### Is Docker Scout included with my Docker Subscription?
+### Does Docker offer academic pricing?
 
-If you have a Docker subscription, you automatically have access to a Docker Scout Free subscription. This lets you enable Docker Scout on up to 3 repositories. To enable Docker Scout for additional repositories, you can [upgrade your plan](../billing/scout-billing.md).
+Contact the [Docker Sales Team](https://www.docker.com/company/contact).
+
+### What ways can I contribute to Docker content?
+
+Docker offers two programs:
```

Comment:
There's also the developer preview program: https://www.docker.com/community/get-involved/developer-preview/

And the feature EAP ones. Maybe we should call those out too?